test/WebJobs.Script.Tests/TestScripts/CSharp/AssembliesFromSharedLocation/run.csx

```diff
@@ -0,0 +1,9 @@
+﻿#r "..\SharedAssemblies\PrimaryDependency.dll"
```

Comment:
Removed the binaries. The assemblies are now dynamically generated as part of the test run.